package.json

```diff
@@ -16,23 +16,19 @@
   "bugs": {
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/webdriverio/webdriverio/blob/master/LICENSE-MIT"
-    }
-  ],
+  "license": "LicenseRef-LICENSE-MIT",
   "main": "./index.js",
   "engines": {
     "node": ">= 0.8.0"
   },
   "scripts": {
-    "test": "node ./test/runner.js",
-    "test-mobile": "_APP=safari _BROWSER=safari _PLATFORMVERSION=7.1 _APPIUMVERSION=1.2 _PLATFORMNAME=iOS _DEVICENAME=iPhone_Simulator _PORT=4723 _ENV=mobile node ./test/runner.js",
-    "test-desktop": "_ENV=desktop _BROWSER=chrome node ./test/runner.js",
-    "test-functional": "_ENV=functional _BROWSER=phantomjs node ./test/runner.js",
-    "test-multibrowser": "_ENV=multibrowser node ./test/runner.js",
+    "test": "grunt test-all",
+    "test-mobile": "grunt test-mobile",
+    "test-desktop": "grunt test-desktop",
+    "test-functional": "grunt test-functional",
+    "test-multibrowser": "grunt test-multibrowser",
     "coverage": "./node_modules/.bin/istanbul cover -x \"**/helpers/_*.js\" ./test/runner.js",
```

Comment:
Correct. test/runner.js has been split into test/setup.js, tes/helper.js and various config in the Gruntfile.js so now all the test targets are run via grunt. See commit logs [c67d0b2](https://github.com/AndreTheHunter/webdriverio/commit/c67d0b2249b101df7794e600916110a5118a9614).
A nice feature of npm is we can run the grunt-cli without needing to have grunt installed globally! This makes it easier to run the tests, people/CI just have to run `npm test-functional`.